src/screens/dashboard/dashboard.tsx

```diff
@@ -44,23 +44,24 @@ const SCREEN_WIDTH = Dimensions.get('window').width;
 const SCROLL_CARD_WIDTH = Math.round(SCREEN_WIDTH * 0.5);
 const ANIMATED_BC_SELECTION = true;
 
-const calculateBalances = (accounts: IAccountState[]) =>
-    accounts.reduce(
+const calculateBalances = (accounts: IAccountState[]) => {
+    return accounts.reduce(
         (out: any, account: IAccountState) => {
             if (!out.balance[account.blockchain]) {
                 out.balance[account.blockchain] = {
-                    amount: account?.balance?.value || new BigNumber(0)
+                    amount: new BigNumber(account?.balance?.value) || new BigNumber(0)
```

Comment:
instead of doing this in a lot of places we need to fix the core problem. We need to somehow restore correct values in redux.